library/src/main/java/com/dslplatform/json/DslJson.java

```diff
@@ -282,7 +282,7 @@
 		 */
 		public Settings<TContext> includeServiceLoader() {
 			withServiceLoader = true;
-			for (Configuration c : ServiceLoader.load(Configuration.class)) {
+			for (Configuration c : ServiceLoader.load(Configuration.class, getClass().getClassLoader())) {
```

Comment:
Hm, this might fix the loading within annotation processor, but looks like it will break it on other use cases?
Probably better to add `includeServiceLoader(ClassLoader)` overload